main.sh

```diff
@@ -213,7 +276,8 @@ for tool in "${tools[@]}"; do
             retry curl --proto '=https' --tlsv1.2 -fsSL --retry 10 --retry-connrefused "${url}" \
                 | tar xzf - -C ${CARGO_HOME:-~/.cargo}/bin
             ;;
-        *) bail "unsupported tool '${tool}'" ;;
+        cargo-binstall) install_cargo_binstall ;;
+        *) cargo_binstall "$tool" "$version" ;;
     esac
 
     info "${tool} installed at $(type -P "${tool}")"
```

Comment:
This line and subsequent version checks will result in an error if the binary name differs from the package name. If the package was installed using cargo-binstall, these should be skipped. (Also, the cargo-binstall displays logs that are equivalent to these messages)

Reply:
I just removed these lines.
If you think there's a better fix, please post another RFC.